EIPS/eip-5920.md

```diff
@@ -42,6 +42,7 @@ Currently, to send ether to an address requires you to call into that address, w
 
 A new opcode is introduced: `PAY` (`0xfc`), which:
 
+- Halt with exceptional failure if the current frame is in `static-mode`.
```

Comment:
This diction matches EIP-7069 and so this version of the text should be approved.